ERROR - 2023-05-18 01:12:05 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: NO) in D:\Workspace\LabMVC4\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 D:\Workspace\LabMVC4\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'dbform', 3306, '', 0)
#1 D:\Workspace\LabMVC4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 D:\Workspace\LabMVC4\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 D:\Workspace\LabMVC4\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 D:\Workspace\LabMVC4\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 D:\Workspace\LabMVC4\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 D:\Workspace\LabMVC4\app\Controllers\Pages.php(69): CodeIgniter\BaseModel->findAll()
#7 D:\Workspace\LabMVC4\system\CodeIgniter.php(934): App\Controllers\Pages->renderDB()
#8 D:\Workspace\LabMVC4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pages))
#9 D:\Workspace\LabMVC4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\Workspace\LabMVC4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 D:\Workspace\LabMVC4\system\Commands\Server\rewrite.php(47): require_once('D:\\Workspace\\La...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: NO) in D:\Workspace\LabMVC4\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 D:\Workspace\LabMVC4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 D:\Workspace\LabMVC4\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 D:\Workspace\LabMVC4\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 D:\Workspace\LabMVC4\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 D:\Workspace\LabMVC4\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 D:\Workspace\LabMVC4\app\Controllers\Pages.php(69): CodeIgniter\BaseModel->findAll()
#6 D:\Workspace\LabMVC4\system\CodeIgniter.php(934): App\Controllers\Pages->renderDB()
#7 D:\Workspace\LabMVC4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pages))
#8 D:\Workspace\LabMVC4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\Workspace\LabMVC4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 D:\Workspace\LabMVC4\system\Commands\Server\rewrite.php(47): require_once('D:\\Workspace\\La...')
#11 {main}
CRITICAL - 2023-05-18 01:12:05 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Pages.php(69): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Pages->renderDB()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pages))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('D:\\Workspace\\LabMVC4\\public\\index.php')
